app/models/user_role.rb

```diff
@@ -34,12 +34,9 @@ def self.normal_user_role
         ProjectPermissions::READ,
         ProjectPermissions::EXPERIMENTS_CREATE,
         ProjectPermissions::COMMENTS_CREATE,
-        ExperimentPermissions::READ,
         ExperimentPermissions::MANAGE,
-        ExperimentPermissions::ARCHIVE,
-        ExperimentPermissions::RESTORE,
-        ExperimentPermissions::CLONE,
-        ExperimentPermissions::TASKS_CREATE,
+        ExperimentPermissions::TASKS_MANAGE,
+        ExperimentPermissions::USERS_MANAGE,
```

Comment:
Review these changes, I think they are not needed anymore

Reply:
Yes, I will review this branch again after yours is merged.